src/main/flight/interpolated_setpoint.c

```diff
@@ -89,11 +91,18 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         if (type == FF_INTERPOLATE_ON) {
             setpointDelta[axis] = setpointDeltaImpl[axis];
         } else {
-            setpointDelta[axis] = 0.5f * (setpointDeltaImpl[axis] + prevSetpointDeltaImpl[axis]);
-            prevSetpointDeltaImpl[axis] = setpointDeltaImpl[axis];
+            if (type == FF_INTERPOLATE_AVG4) {
```

Comment:
This can be done with a `laggedMovingAverage` filter:

https://github.com/betaflight/betaflight/blob/4433dbb5186f0568f9781485928a119557d65552/src/main/common/filter.c#L206-L227

Reply:
Would it likely be more efficient to invoke that filter?

Reply:
Yes - the filter avoids shuffling around of the previous values.